### ISSUE TITLE
Resizable columns

### ISSUE THREAD

**Issue by techniq** [techniq/mui-table#5]
- [React-Table - HOC DraggableTable](https://codesandbox.io/s/5vxlnjrw1n)
- [react-virtualized-resizable-columns](https://codesandbox.io/s/j30k46l7xw)
- https://github.com/mzabriskie/react-draggable
- http://clauderic.github.io/react-sortable-hoc/#/react-virtualized/sortable-table-columns?_k=o5p3dd

**Comment by ericpyle** [comment]
@techniq I just realized that I need this feature, probably moreso than other things. I suppose I could wrap the contents in a Tooltip to show the full content if the column width hides some info?

**Comment by techniq** [comment]
@ericpyle I have used tooltips in the past, or allowed controlling which columns are available to see (to give more room for columns you care about).

I should be pushing a beta of the non-react virtualized version soon (possibly later today).  This will allow for `display: table` layouts (and `display: grid` at some point) which can more easily wrap long columns or size themselves to fit contents, which is difficult with react-virtualized's `Grid`.   I think this change alone will help reduce the need for the feature (but not remove), but I still plan to support it in some vain at some point (leveraging react-draggable likely).

**Comment by ericpyle** [comment]
@techniq yes, I think warpping or auto-sizing to fit contents would probably be sufficient in most cases. Thanks for working on that!

**Comment by techniq** [comment]
@ericpyle Version 2.0.0 released - https://github.com/techniq/mui-table/issues/13
